pop-api/integration-tests/src/local_fungibles.rs

```diff
@@ -263,15 +260,10 @@ fn token_decimals_asset(asset_id: AssetId) -> u8 {
 /// - decrease_allowance
 
 #[test]
-#[ignore]
 fn total_supply_works() {
 	new_test_ext().execute_with(|| {
 		let _ = env_logger::try_init();
-		let addr = instantiate(
-			"../../pop-api/examples/fungibles/target/ink/fungibles.wasm",
-			INIT_VALUE,
-			vec![],
-		);
+		let addr = instantiate("contracts/fungibles/target/ink/fungibles.wasm", INIT_VALUE, vec![]);
```

Comment:
IMO, these .wasm files should be moved to an `artifacts` or `fixtures` directory. Otherwise these tests may fail if someone hasn't built all of the contracts.

Reply:
Included in #109